Libraries/PermissionsAndroid/PermissionsAndroid.js

```diff
@@ -75,7 +75,7 @@ const PERMISSIONS = Object.freeze({
   ANSWER_PHONE_CALLS: 'android.permission.ANSWER_PHONE_CALLS',
   READ_PHONE_NUMBERS: 'android.permission.READ_PHONE_NUMBERS',
   UWB_RANGING: 'android.permission.UWB_RANGING',
-  POST_NOTIFICATION: 'android.permission.POST_NOTIFICATIONS',
+  POST_NOTIFICATIONS: 'android.permission.POST_NOTIFICATIONS',
```

Comment:
Thanks for spotting this. I'm in for a breaking change. The problem is that we shipped this in 0.70 as well, so we'll need to backport this.

Potentially having both is a preferred approach and in 0.72 we can remove the duplicate? (cc @kelset for visiblity)

Reply:
I could add `console.warn` whenever `POST_NOTIFICATION` is used if that's what we want and keep them both for 0.71.

Reply:
making sure that I'm following: you want to merge this in main and then cherry pick for both 0.71 and 0.70? I'm ok with that, let's make sure to post comments with links in both release discussions

Reply:
Asking for your opinion as this is essentially a 'typo' we let pass through. We can add both `POST_NOTIFICATION` and `POST_NOTIFICATIONS` in the JS API and have the breaking change in .72 OR have this as a breaking change in both .70 and .71

Reply:
that said, my personal preference would be no breaking changes in 0.70 and 0.71, but yes for 0.72

Reply:
Ok so let's merge this and pick it on 0.70/0.71
Then let's do the breaking change on main for 72